content/md/first-steps/manage-data-quality.md

```diff
@@ -0,0 +1,30 @@
+---
+id: manage-data-quality
+themes: catalog-settings
+title: Manage your **data quality**
+popular: false
+ee-only: true
+related: understand-data-quality, improve-data-quality
+---
+
+# Overview
+
+In order to check, score and get recommendations on the product data that truly matters to craft a great product experience,  you can disable Data Quality Insights calculation for non relevant attribute groups (ie. ERP, internal…)
+By doing so, you will be able to highlight the quality of the attributes that are the most valuable for your customers.
+
+# Data quality insights activation and deactivation
+
+:::info
+You must be granted a specific permission by the administrator to be able to edit attribute groups.
+:::
+
+To deactivate data quality insights from an attribute group:
+1.  Go to `Settings/Attribute groups`
+2.  Click on an `Attribute group` name
+3.  Set `Data Quality activation` to No
+
+ [disable attribute group from dqi](dqi-disablement-attribute-group.html)
```

Comment:
Why is there a link on nothing here? That's weird. :)

Reply:
yes, that was my purpose. I'll remove it for now